File: 17- স্বরবর্ন গণনা.c
#include<stdio.h>
int main()
{

    int t;
    scanf("%d",&t);
    char s[1000];
    for(int i=0;i<t;i++)
    {
        scanf(" %[^\n]",s);
        int c=0;
        for(int j=0;s[j]!='\0';j++)
        {
            if(s[j]=='a'||s[j]=='e'||s[j]=='i'||s[j]=='o'||s[j]=='u'||s[j]=='A'||s[j]=='E'||s[j]=='I'||s[j]=='O'||s[j]=='U')
                c++;


        }
        printf("Number of vowels = %d\n",c);

    }

    return 0;


}

File: 17- স্বরবর্ন গণনা.py
t = int(input())

while t!=0:
    s = input()
    c = 0
    for i in range(len(s)):
        if s[i] == 'a' or s[i] == 'e' or s[i] == 'i' or s[i] == 'o' or s[i] == 'u' or s[i] == 'A' or s[i] == 'E' or s[i] == 'I' or s[i] == 'O' or s[i] == 'U':
            c+=1
    t-=1
    print("Number of vowels =",c)
